packages/now-cli/test/dev-server.unit.js

```diff
@@ -498,3 +498,12 @@ test('[DevServer] parseListen()', t => {
   t.truthy(err);
   t.is(err.message, 'Unknown `--listen` scheme (protocol): bad:');
 });
+
+test(
+  '[DevServer] index.html prioritized over other index.* files',
+  testFixture('now-dev-prioritize-index-html', async (t, server) => {
+    const res = await fetch(`${server.address}`);
+    const html = await res.text();
+    t.is(html.includes(`<title>Hello World</title>`) > -1, true);
```

Comment:
```suggestion
    t.truthy(html.includes('<title>Hello World</title>'));
```

`.includes` already returns a boolean